recipes/arrow/all/conandata.yml

```diff
@@ -21,14 +21,11 @@ patches:
   "10.0.0":
     - patch_file: "patches/10.0.0-0001-mallctl-takes-size_t.patch"
       patch_description: "use size_t instead of ssize_t"
-      patch_type: "backport"
-    - patch_file: "patches/10.0.0-0002-fix-cmake.patch"
-      patch_description: "use cci package"
       patch_type: "conan"
   "8.0.1":
     - patch_file: "patches/8.0.0-0003-mallctl-takes-size_t.patch"
       patch_description: "use size_t instead of ssize_t"
-      patch_type: "backport"
+      patch_type: "conan"
```

Comment:
These `patch_type` changes are only for fixing lint error: https://github.com/conan-io/conan-center-index/actions/runs/3764042096/jobs/6398083308

Should we create a separate pull request for them?

Reply:
@kou You can add to the same PR, no problem. Thank you for spotting it!